extensions/uniqueForm.ajax.js

```diff
@@ -16,11 +16,15 @@
 				uniqueForm.formSubmitBeforeHandler.apply(uniqueForm, [$form]);
 				setTimeout(function () {
 					//disable je třeba nastavit opožděně, aby formulář odeslal v POST requestu i submit button
-					$form.find(uniqueForm.inputSubmitSelector).each(uniqueForm.inputSubmitBeforeHandler);
+					$form.find(uniqueForm.inputSubmitSelector).each(function () {
+						uniqueForm.inputSubmitBeforeHandler.apply(this, [uniqueForm.buttonClassDisabled]);
```

Comment:
Proč se v tomto místě předává název class jako parametr? Proč ho rovnou nepoužít v tom callbacku jako `this.buttonClassDisabled`?

Reply:
Zeptaj se pak Vítě, ja to jen styloval a on si pak řekl jak to mám commitnout a samozřejmě to v tom js i napsal.